std/container/binaryheap.d

```diff
@@ -25,6 +25,24 @@ import std.traits;
 
 public import std.container.util;
 
+///
+unittest
+{
+    import std.algorithm : equal;
```

Comment:
isn't equal in std.algorithm.comparison ? I mean if you use selected imports, use them on deepest nested module.